### ISSUE TITLE
Proposal: Allow and enforce trailing commas

### ISSUE THREAD

**Issue by cameron-martin** [ForgeRock/eslint-config#4]
I find trailing commas on all multiline things (function arguments, arrays & object literals) handy when moving lines around and merging. I haven't found any downsides to this.

See [the comma-dangle eslint rule](https://eslint.org/docs/rules/comma-dangle).

**Comment by juliankigwana** [comment]
Personally I prefer without, but I guess you're the main man now, so you can change to what you like ;)

**Comment by philostler** [comment]
@juliankigwana 🤣  custom eslint configs are the dark ages now. can't believe we used to waste time on this.

**Comment by juliankigwana** [comment]
> @juliankigwana 🤣 custom eslint configs are the dark ages now. can't believe we used to waste time on this.

That is true now, however, needs change. At the time there were sparse alternatives, and we needed to bring our code styles together.

**Comment by jamesphillpotts-fr** [comment]
@philostler @juliankigwana move along please, nothing to see here.
